apps/users-service/src/users-service.service.ts

```diff
@@ -90,7 +90,7 @@ export class UsersService {
           )
       )
 
-      if (this.configService.get<string>('NODE_ENV') === 'FEAUTRE_YET_ENABLED') {
+      if (this.configService.get<string>('NODE_ENV') === 'PRODUCTION') {
```

Comment:
we don't want casing mismatch here. check if NODE_ENV !== undefined then lowercase it to match a lowercase "production"